include/wx/textctrl.h

```diff
@@ -555,7 +555,8 @@ class WXDLLIMPEXP_CORE wxTextAreaBase
     virtual int GetLineLength(long lineNo) const = 0;
     virtual wxString GetLineText(long lineNo) const = 0;
     virtual int GetNumberOfLines() const = 0;
-
+    virtual unsigned GetPhysicalLineCount() const;
+    virtual unsigned GetLogicalLineCount() const;
```

Comment:
That was my initial idea, but because then wxRTC and wxSTC will need the implementation, it was abandoned.
And I don't know their code well enough to submit them for those controls.

Reply:
Placing declaration here (in wxTextAreaBase) means these two methods will be present in everything that derives from wxTextAreaBase or wxTextCtrlIface: That includes wxRichtTextCtrl, which would do better with more distinction (paragraph end vs. line break), and wxSearchCtrlBaseBaseClass which is never multiline.

Wouldn't it be better to move these declarations into wxTextCtrl() class? Then, they could be declared pure virtual and implementations would be needed only in wxTextCtrl control for every platform. Interface can be added to other controls, when someone feels like actually implementing it.